app/javascript/admin/controllers/article_controller.js

```diff
@@ -1,7 +1,8 @@
 import { Controller } from 'stimulus';
 
 export default class ArticleController extends Controller {
-  static targets = ['featuredNumber'];
+  static classes = ['bgHighlighted', 'borderHighlighted'];
```

Comment:
classes allow the developer not to hardcode the CSS class name in the Stimulus controller, see https://stimulusjs.org/reference/css-classes